docsbox/docs/tasks.py

```diff
@@ -78,12 +79,15 @@ def process_document_convertion(path, options, meta, current_task):
             force_ocr += 1
     else:
         with TemporaryDirectory() as tmp_dir:
```

Comment:
Guess we don't need this temp dir now.

Is the configuration file included at all in the request any longer? Is it necessary to include it anymore if we don't need to edit any other configuration from there? Or should we just keep it included for future's sake if we ever need to configure anything?

Reply:
the temp dir is for a the env:UserInstallation this allows to assign each instance its own environment useful for running instances in parallel. I think we should keep the config file since there are other configurations there that could be change if needed be, so we dont have to add every config in the export filter